common/app/common/commercial/hosted/HostedArticlePage.scala

```diff
@@ -10,7 +10,25 @@ case class HostedArticlePage(
   pageName: String,
   pageTitle: String,
   standfirst: String,
-  facebookImageUrl: String
+  standfirstLink: String,
```

Comment:
Great work @Calanthe ! Forget what I said, as per @kelvin-chappell 's comment, this is probably part of your next itration.